content/en/docs/concepts/extend-kubernetes/compute-storage-net/resource-drivers.md

```diff
@@ -0,0 +1,286 @@
+---
+title: DRA Resource Drivers
+description: Resource drivers provide non-trivial allocation logic and management for devices or resources that require vendor-specific or just complex setup, such as GPUs, NICs, FPGAs, etc.
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+
+Kubernetes provides a
+[Dynamic Resource Allocation (DRA)](https://kubernetes.io/docs/concepts/scheduling-eviction/dynamic-resource-allocation/)
+that can be leveraged to provide more complex hardware resources to workloads with custom resource
+accounting.
+
+Similarly to {{< glossary_tooltip term_id="device-plugin" text="Device Plugins">}}, instead of
+customizing the code for Kubernetes itself, vendors can implement a _resource driver_ that you deploy
+into the cluster to account for and control the allocation of GPUs, high-performance NICs, FPGAs,
+InfiniBand adapters, and other similar computing resources that may require vendor specific
+initialization and setup.
+
+With device plugins, the scheduler was given a trivial, numerical representation of the resources
+available on node for consideration during scheduling as an extended resource.
+
+With DRA, the scheduler is offloading the task of allocating and accounting for non-native resources
+to the resource driver, which manages such resources in the cluster.
+
+Resource driver consists of two main components:
+
+- a _controller_ (one per cluster), manages hardware resources allocation for
+  {{< glossary_tooltip term_id="resource-claim" text="ResourceClaims">}}
+- _kubelet-plugin_ (one per node with resource), that:
```

Comment:
```suggestion
- A _kubelet plugin_ (deployed on each node that can access the associated resource), that:
```

Reply:
Actually, this is relevant too, since the resource doesn't have to be local. I'll combine both phrasings.

Reply:
Done.